app/src/main/java/org/schabi/newpipe/fragments/list/search/SearchFragment.java

```diff
@@ -845,7 +843,8 @@ public void startLoading(final boolean forceLoad) {
 
     @Override
     protected void loadMoreItems() {
-        if (nextPageUrl == null || nextPageUrl.isEmpty()) {
+        if (nextPage == null || ((nextPage.getUrl() == null || nextPage.getUrl().isEmpty())
```

Comment:
same here

Reply:
What about creating a static method in the Page class to replace this duplicate piece of code?